### PR TITLE
bcm53xx: Add support for Arris SBR-AC1900P

### PR DESCRIPTION
**Pull request by paldier**
This router has BCM4709A0 SoC, 128 MiB NAND flash (MX30LF1G18AC-TI),
256 MiB memory and 4 x LAN and 1 x WAN ports.
WiFi chips are BCM4360 x 2.
USB2.0 x 1. USB3.0 x 1.
this router cpu port only used port 8,but b53 must set port 7, is it a bug?
see 700-b53-add-hacky-CPU-port-fixes-for-devices-not-using-p.patch

### CODE REVIEW

File: target/linux/bcm53xx/base-files/etc/board.d/02_network [hunk 1]
```diff
@@ -38,6 +38,11 @@ asus,rt-ac87u)
 	ifname=eth1
 	etXmacaddr=$(nvram get et1macaddr)
 	;;
+arris,sbr-ac1900p | \
+arris,sbr-ac3200p)
```
Comment (hauke) [inline target/linux/bcm53xx/base-files/etc/board.d/02_network]: arris,sbr-ac3200p should probably go into the other pull request

File: target/linux/bcm53xx/patches-4.14/322-ARM-dts-BCM5301X-Add-DT-for-ARRIS-SBR-AC1900P.patch [hunk 2]
```diff
@@ -0,0 +1,118 @@
...
+ 	bcm47094-luxul-abr-4500.dtb \
+--- /dev/null
++++ b/arch/arm/boot/dts/bcm4709-arris-sbr-ac1900p.dts
+@@ -0,0 +1,97 @@
++// SPDX-License-Identifier: GPL-2.0-or-later OR MIT
++/*
++ * DTS for ARRIS SBR-AC1900P
++ *
++ * Copyright (C) 2019 lostlonger <lostlonger.g@gmail.com>
++ * Copyright (C) 2019 paldier <paldier@hotmail.com>
```
Comment (hauke) [inline target/linux/bcm53xx/patches-4.14/322-ARM-dts-BCM5301X-Add-DT-for-ARRIS-SBR-AC1900P.patch]: Please use a real name here

File: target/linux/bcm53xx/patches-4.14/322-ARM-dts-BCM5301X-Add-DT-for-ARRIS-SBR-AC1900P.patch [hunk 3]
```diff
@@ -0,0 +1,118 @@
+This router has BCM4709A0 SoC, 128 MiB NAND flash (MX30LF1G18AC-TI),
+256 MiB memory and 4 x LAN and 1 x WAN ports.
+WiFi chips are BCM4360 x 2.
+USB2.0 x 1. USB3.0 x 1.
+this router cpu port only used port 8,but b53 must set port 7, is it a bug?
+see 700-b53-add-hacky-CPU-port-fixes-for-devices-not-using-p.patch
```
Comment (hauke) [inline target/linux/bcm53xx/patches-4.14/322-ARM-dts-BCM5301X-Add-DT-for-ARRIS-SBR-AC1900P.patch]: Probably both ports are connected between the Switch and the SoC.

**Review by hauke — COMMENTED** [review]
Please add a signed-off-by line to the patch.
You can probably also squash both commit together.

**Comment by paldier** [comment]
Thanks,i have updated

File: target/linux/bcm53xx/patches-4.14/322-ARM-dts-BCM5301X-Add-DT-for-ARRIS-SBR-AC1900P.patch (hunk 3 above)
Comment (paldier) [reply target/linux/bcm53xx/patches-4.14/322-ARM-dts-BCM5301X-Add-DT-for-ARRIS-SBR-AC1900P.patch]: I guess this is difference between b53 and et

File: target/linux/bcm53xx/image/Makefile [hunk 4]
```diff
@@ -320,6 +320,12 @@ define Device/phicomm-k3
 endef
 TARGET_DEVICES += phicomm-k3
 
+define Device/arris-sbr-ac1900p
```
Comment (adrianschmutzler) [inline target/linux/bcm53xx/image/Makefile:323]: Alphabetic sorting.

**Comment by adrianschmutzler** [comment]
This needs to be updated to 4.19/5.4. However, there hasn't been a reviewer for one year, so I'm not sure whether it's really worth it. If this is not updated during the next month (due to inactivity or because it's not worth it, I will close the PR).

**Pull request closed without merging**